### PR TITLE
New Term Entry: PHP String Functions, `substr()`

### PR DESCRIPTION
**Pull request by SSwiniarski**
### Description

Add new term for PHP string functions, `substr()`. From the Target Keyword list. Assumes the directory structure from merging  [660](https://github.com/Codecademy/docs/pull/660)

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/php/concepts/string-functions/terms/substr/substr.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+## Syntax
+
+```php
+substr($string, $offset, $length)
+```
+
+The `$string` parameter is the string we're returning a portion of.
+
+The `$offset` parameter is an `int` value ad specifies the start of the substring in the following manner:
```
Comment (KTom101) [inline content/php/concepts/string-functions/terms/substr/substr.md:26]:
```suggestion
The `$offset` parameter is an `int` value and specifies the start of the substring in the following manner:
```

**Review by KTom101 — CHANGES_REQUESTED** [review]
@SSwiniarski I'm requesting a couple of minor changes.

**Comment by SSwiniarski** [comment]
@KTom101, I applied your suggestions.

**Review by KTom101 — APPROVED** [review]
@SSwiniarski Looks ready for the next review.

File: content/php/concepts/string-functions/terms/substr/substr.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The `substr()` function returns a portion of a string specified by parameters for offset and length.
+
+## Syntax
+
+```php
```
Comment (Dusch4593) [inline content/php/concepts/string-functions/terms/substr/substr.md:20]:
```suggestion
```pseudo
```

File: content/php/concepts/string-functions/terms/substr/substr.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+substr($string, $offset, $length)
+```
+
+The `$string` parameter is the string from which a portion will be returned.
+
+The `$offset` parameter is an `int` value and specifies the start of the substring in the following manner:
+
+- Positive or zero values specify the zero-based character position from the start of the string.
+- Negative values indicate the character position from the end of the string.
+- An `$offset` greater than the length of `$string` will return an empty string.
```
Comment (Dusch4593) [inline content/php/concepts/string-functions/terms/substr/substr.md:30]:
```suggestion
- A value greater than the length of `$string` will return an empty string.
```

File: content/php/concepts/string-functions/terms/substr/substr.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+The `$length` parameter is an optional `int` that specifies the number of characters to return in the following manner:
+
+- Positive values specify to return that number of characters, or characters up to the end of `$string` if fewer than `$length` characters are left after `$offset` is determined.
+- A `null` or omitted value will return characters from `$offset` until the end of `$string`.
+- Negative values will return characters from `$offset` until `$length` characters from the end of `$string`.
+- A value of zero, or a negative value that specifies a position before `$offset`, will return an empty string.
+
+## Codebyte Example
+
```
Comment (Dusch4593) [inline content/php/concepts/string-functions/terms/substr/substr.md:40]: Let's add a brief sentence here that describes what's going on in the codebyte below.

**Comment by SSwiniarski** [comment]
@Dusch4593, applied your changes.

**Review by Dusch4593 — APPROVED** [review]
@SSwiniarski LGTM! 👍🏻

**Comment by SSwiniarski** [comment]
@Dusch4593, @KTom101 do we look ok to merge?

**Pull request merged**